_source/_posts/2018-06-28-tutorial-build-a-basic-crud-app-with-node.md

```diff
@@ -55,76 +62,48 @@ npm install -g express-generator
-
-{% img blog/tutorial-build-a-basic-crud-app-with-node/okta-signup.png alt:"Okta signup page" width:"700" %}{: .center-image }
-
-Once you've logged in and land on the dashboard page, copy down the **Org URL** pictured below. You will need this later.
-
-{% img blog/tutorial-build-a-basic-crud-app-with-node/okta-org-url.png alt:"Okta Org URL" width:"700" %}{: .center-image }
-
-Then create a new application by browsing to the **Applications** tab and clicking **Add Application**.
-
-{% img blog/tutorial-build-a-basic-crud-app-with-node/okta-app-dashboard.png alt:"Okta app dashboard" width:"700" %}{: .center-image }
-
-Next, click the **Web** platform option (since our blog project is a  web app).
-
-{% img blog/tutorial-build-a-basic-crud-app-with-node/okta-create-app-platform.png alt:"Okta create app platform" width:"700" %}{: .center-image }
-
-On the settings page, enter the following values:
-
-- **Name**: Blog
-- **Base URIs**: `http://localhost:3000`
-- **Login redirect URIs**: `http://localhost:3000/users/callback`
+Dealing with user authentication in web apps can be a massive pain for every developer. This is where Okta shines: it helps you secure your web applications with minimal effort. 
 
-You can leave all the other values unchanged.
+{% include setup/cli.md type="web" loginRedirectUri="http://localhost:3000/authorization-code/callback" logoutRedirectUri="http://localhost:3000/" %}
 
-{% img blog/tutorial-build-a-basic-crud-app-with-node/okta-create-app-settings.png alt:"Okta create app settings" width:"700" %}{: .center-image }
-
-Now that your application has been created, copy down the **Client ID** and **Client secret** values on the following page, you'll need them soon.
-
-{% img blog/tutorial-build-a-basic-crud-app-with-node/okta-app-secrets.png alt:"Okta app secrets" width:"700" %}{: .center-image }
-
-Finally, create a new authentication token. This will allow your app to talk to Okta to retrieve user information, among other things. To do this, click the **API** tab at the top of the page followed by the **Create Token** button. Give your token a name, preferably the same name as your application, then click **Create Token**. Copy down this token value as you will need it soon.
-
-{% img blog/tutorial-build-a-basic-crud-app-with-node/okta-create-token.png alt:"Okta create token" width:"700" %}{: .center-image }
+Finally, create a new API token. This will allow your app to talk to Okta to retrieve user information, among other things. You can find the steps to create an API token [here](https://developer.okta.com/docs/guides/create-an-api-token/main/). Copy down this token value as you will need it soon.
```

Comment:
```suggestion
Finally, create a new API token. This will allow your app to talk to Okta to retrieve user information, among other things. You can find the steps to [create an API token here](https://developer.okta.com/docs/guides/create-an-api-token/main/). Copy down this token value as you will need it soon.
```
Here's the guideline from our internal InfoDev style guide: "Never make a link out of words like here or this page.... " - https://oktawiki.atlassian.net/wiki/spaces/DOC/pages/722929640/InfoDev+Style+Guide#1.-Guidance-on-writing-and-style. - I try to make the links descriptive of what the reader will find when they click.